File: Transpose of Matrix[10-july-2023]/solution.py
#  >> PYTHON CODE <<

class Solution:
    def transpose(self, matrix, n):
        for i in range(n):
            for j in range(i+1,n):
                #swap matrix[i][j] and matrix[j][i]
                matrix[i][j],matrix[j][i]=matrix[j][i],matrix[i][j]

File: Transpose of Matrix[10-july-2023]/Solution.java
//  >> JAVA CODE <<

class Solution
{
    public void transpose(int n,int a[][])
    {
        for(int i=0;i<n;i++)
        {
            for(int j=i+1;j<n;j++)
            {
                int temp=a[i][j];
                a[i][j]=a[j][i];
                a[j][i]=temp;
            }
        }
    }
}